### ISSUE TITLE
meteor.com is gone, so your demos no longer work.

### ISSUE THREAD

**Issue by blairn** [lookback/meteor-dropdowns#13]
if you could host them somewhere else, that would be grand :)

**Comment by johanbrook** [comment]
Yep, I've received emails from MDG about it. Will need to figure something out.

**Comment by zachsa** [comment]
Still down :(

**Comment by ivansglazunov** [comment]
Very convenient working with herokuapp. Independently expands repository. Many free projects.
